openquakeplatform/set_thumb/mapthumbnail_test.py

```diff
@@ -25,16 +28,44 @@ def set_thumbs_map_test(self):
         for link in links:
             pla.get('/maps/%s' % link)
 
+            time.sleep(3)
+
```

Comment:
are still required this delays ?

Reply:
yes because need to attend the loading of page